### ISSUE TITLE
iOS 17 Support Update

### ISSUE THREAD

**Issue by riei2922jww** [master131/iFakeLocation#169]
Hello,

I know you have had this topic raised multiple times but please can you provide an update on when and if there will be an update for iOS 17?

I love this tool so much, I really hope you are able to successfully update it soon, thank you :)

**Comment by Prabh-Saini** [comment]
#141 it will be fixed when someone creates an API that allows Windows to communicate with the iPhone over a specific 'bridge'. It is possible that Linux can be used in the future but at the moment hold on tight. Realistically we are a very long away from a solution, your best bet is using an alternative older device running iOS 16 or older. Like an iPhone 6/7/8/X.

**Comment by davesc63** [comment]
For windows iOS17 support, check out https://geoport.me

**Comment by riei2922jww** [comment]
> For windows iOS17 support, check out https://geoport.me

Hey, thank you so much, is this a new thing? Never heard of it and been searching for ages, who is the developer? :)

**Comment by davesc63** [comment]
> > For windows iOS17 support, check out https://geoport.me
> 
> Hey, thank you so much, is this a new thing? Never heard of it and been searching for ages, who is the developer? :)

Developer is me haha. 
im a fan of iFakeLocation and used it for ages until iOS17 stopped working. 

I then found the pymobiledevice3 project and set about on a personal project to make my own version to spoof location for a few apps

Have had it working as a cli tool (fuel-spoof) and now the gui version for around 2 months. Now looking to share it with others :)

**Comment by riei2922jww** [comment]
> > > For windows iOS17 support, check out https://geoport.me
> 
> > 
> 
> > Hey, thank you so much, is this a new thing? Never heard of it and been searching for ages, who is the developer? :)
> 
> 
> 
> Developer is me haha. 
> 
> im a fan of iFakeLocation and used it for ages until iOS17 stopped working. 
> 
> 
> 
> I then found the pymobiledevice3 project and set about on a personal project to make my own version to spoof location for a few apps
> 
> 
> 
> Have had it working as a cli tool (fuel-spoof) and now the gui version for around 2 months. Now looking to share it with others :)

Hi, it's really nice and I'm super grateful you've put this together but it's asking me to download external "Tencent" drivers? I already have the Apple drivers? How do I just use these ones instead of downloading externally? Thanks.

**Comment by davesc63** [comment]
> > > > For windows iOS17 support, check out https://geoport.me
> > 
> > 
> > > 
> > 
> > 
> > > Hey, thank you so much, is this a new thing? Never heard of it and been searching for ages, who is the developer? :)
> > 
> > 
> > Developer is me haha.
> > im a fan of iFakeLocation and used it for ages until iOS17 stopped working.
> > I then found the pymobiledevice3 project and set about on a personal project to make my own version to spoof location for a few apps
> > Have had it working as a cli tool (fuel-spoof) and now the gui version for around 2 months. Now looking to share it with others :)
> 
> Hi, it's really nice and I'm super grateful you've put this together but it's asking me to download external "Tencent" drivers? I already have the Apple drivers? How do I just use these ones instead of downloading externally? Thanks.

No. You need to use the patched driver. That’s what allows the iOS17 on Windows. If the native Apple drivers were enough there wouldn’t be an issue supporting iOS17 with current apps

**Comment by vinhnv2507** [comment]
> Để được hỗ trợ iOS17 cho Windows, hãy xem https://geoport.me

@davesc63 Does it support iOS 18?

**Comment by davesc63** [comment]
> > Để được hỗ trợ iOS17 cho Windows, hãy xem https://geoport.me
> 
> [@davesc63](https://github.com/davesc63) Does it support iOS 18?

Yes, iOS 18 and iOS 26 are all confirmed as working with GeoPort
